optics-core/CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# optics-core-0.3 (TBW)
+* GHC-8.10 support
+* Add `filteredBy` and `unsafeFilteredBy`
+* Add indexed instances for `Const` and `Constant`
+* Add `afoldVL` and `iafoldVL` constructors
+* Rename `toAtraversalVL` to `atraverseOf`, same for indexed variant
+* `element` and `elementOf` construct `(Ix)AffineTraversal`
+* Change `mapping` to work also on Lens etc (degenerating to Getter etc.)
```

Comment:
Too many etcS ;) Writing that `mapping` now works for various optic kinds would be imo better.